server/lib/models/maps.js

```diff
@@ -143,6 +143,16 @@ export async function mapExists(hash) {
 }
 
 export async function getMapInfo(mapIds, userId) {
+  const favoritedJoin = userId
```

Comment:
convert this to use the sql template tag thing maybe?

should make the variable number of parameters easier to use, I think :)